frontend/package.json

```diff
@@ -35,5 +41,12 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "@material-ui/icons": "^4.11.2",
```

Comment:
you should only dependencies here that are relevant for dev like prop-types or prettier. Therefore you should move "@material-ui/icons", "@material-ui/styles" and "react-swipeable-views-utils" to the normal dependencies

Reply:
done